charts/elasticsearch/templates/podsecuritypolicy.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.podSecurityPolicy.create -}}
+{{- $fullName := include "elasticsearch.uname" . -}}
+apiVersion: policy/v1beta1
```

Comment:
This needs to change as well: https://kubernetes.io/docs/reference/using-api/deprecation-guide/#psp-v125

https://kubernetes.io/docs/tasks/configure-pod-container/migrate-from-psp/

Reply:
We actually don't use this, so we could leave it as it is.